verilog/formatting/tree_unwrapper.cc

```diff
@@ -1263,6 +1263,32 @@ void TreeUnwrapper::ReshapeTokenPartitions(
       }
     case NodeEnum::kBindDirective: {
       AttachTrailingSemicolonToPreviousPartition(&partition);
+
```

Comment:
Suggestion:
Instead of replicating the logic done in kDataDeclaration, by the time you reach this point, you already have the data declaration subpartition shaped properly.  I think you should be able to "push the `bind`" keyword downward into the leftmost leaf using `MergeLeafIntoNextLeaf`, and be done.  (Code below will hoist the resulting only-child partition to flatten it one level.)

Reply:
Looks much simpler now!

Reply:
Done, thanks for the tip.